src/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,375 @@
+} & SxProp
+
+// Default state for the `visible` prop when a sub component is only visible on narrow viewport
+const hiddenOnRegularAndWide = {
+  narrow: false,
+  regular: true,
+  wide: true,
+}
+
+// Default state for the `visible` prop when a sub component is visible on regular and wide viewport
+const hiddenOnNarrow = {
+  narrow: true,
+  regular: false,
+  wide: false,
+}
+
+// Root
+// -----------------------------------------------------------------------------
+export type PageHeaderProps = {
+  'aria-label'?: React.AriaAttributes['aria-label']
+  as?: React.ElementType | 'header' | 'div'
+} & sharedPropTypes
+
+const Root: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({children, sx = {}, as = 'div'}) => {
+  const rootStyles = {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '0.5rem',
+  }
+  return (
+    <Box as={as} sx={merge<BetterSystemStyleObject>(rootStyles, sx)}>
+      {children}
+    </Box>
+  )
+}
+
+// PageHeader.ContextArea : Only visible on narrow viewports by default to provide user context of where they are at their journey. `visible` prop available
+// to manage their custom visibility but consumers should be careful if they choose to hide this on narrow viewports.
+// PageHeader.ContextArea Sub Components: PageHeader.ParentLink, PageHeader.ContextBar, PageHeader.ContextAreaActions
+// ---------------------------------------------------------------------
+
+const ContextArea: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({
+  children,
+  hidden = hiddenOnRegularAndWide,
+  sx = {},
+}) => {
+  const isHidden = useResponsiveValue(hidden, false)
+  const contentNavStyles = {
+    display: isHidden ? 'none' : 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: '0.5rem',
+    order: REGION_ORDER.ContextArea,
+  }
+  return <Box sx={merge<BetterSystemStyleObject>(contentNavStyles, sx)}>{children}</Box>
+}
+// adopted from React.AnchorHTMLAttributes
+type LinkProps = {
+  download?: string
+  href?: string
+  hrefLang?: string
+  media?: string
+  ping?: string
+  rel?: string
+  target?: string
+  type?: string
+  referrerPolicy?: React.AnchorHTMLAttributes<HTMLAnchorElement>['referrerPolicy']
+}
+export type ParentLinkProps = PageHeaderProps & LinkProps
+
+const ParentLink = React.forwardRef<HTMLAnchorElement, ParentLinkProps>(
+  (
+    {
+      children,
+      sx = {},
+      href,
+      'aria-label': ariaLabel = `Back to ${children}`,
+      as = 'a',
+      hidden = hiddenOnRegularAndWide,
+    },
+    ref,
+  ) => {
+    const isHidden = useResponsiveValue(hidden, false)
+    return (
+      <>
+        <Link
+          ref={ref}
+          as={as}
+          aria-label={ariaLabel}
+          muted
+          sx={merge<BetterSystemStyleObject>(
+            {
+              display: isHidden ? 'none' : 'flex',
+              alignItems: 'center',
+              lineHeight: '2.75rem', // min touch target size 44px (16*2.75)
```

Comment:
I don't actually think we need this. Our buttons don't have a minimum height of 44px, and they're still  considered to be accessible.

Reply:
When I remove this, its line-height becomes 1.5 (inherits from the parent) and would still be okay for accessibility, you think?

Reply:
Yup, I think that's fine 👍